conda/cli/main_clean.py

```diff
@@ -65,6 +72,66 @@ def configure_parser(sub_parsers):
     p.set_defaults(func=execute)
 
 
+# work-around for python bug on Windows prior to python 3.2
```

Comment:
I still have a problem with this whole block of code, but these problems can be fixed...

Step 1, rebase against master to pull in the latest test configuration.
Step 2, turn this whole try/except block into an inner function within `cross_platform_st_nlink`.
Step 3, make the `ctypes` imports inline within `cross_platform_st_nlink` or that new inner function as necessary.
There might be a Step 4, which would be to write some additional tests.  Can wait on that though to see how the current test coverage looks.